v2/api_error.go

```diff
@@ -0,0 +1,185 @@
+package gax
+
+// code snippet credit to @ndietz
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"google.golang.org/genproto/googleapis/rpc/errdetails"
+	"google.golang.org/grpc/status"
+)
+
+//ErrDetails holds the google/rpc/error_details.proto messages.
```

Comment:
Please put a space between `//` and the name, so `// ErrDetails`. Applies to all leading comments.